components/service/glean/src/test/java/mozilla/components/service/glean/DispatchersTest.kt

```diff
@@ -8,20 +8,22 @@ import kotlinx.coroutines.runBlocking
 import org.junit.Test
 
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
 import org.junit.Assert.assertNotSame
 import org.junit.Assert.assertSame
 
+@Suppress("EXPERIMENTAL_API_USAGE")
 class DispatchersTest {
 
     @Test
     fun `API scope runs off the main thread`() {
         val mainThread = Thread.currentThread()
         var threadCanary = false
-        @Suppress("EXPERIMENTAL_API_USAGE")
         Dispatchers.API.setTestingMode(false)
+        Dispatchers.API.queueInitialTasks = false
```

Comment:
Mh, should we have an internal test only method for setting the initial task flag, as we do for the testing mode? at the very least, we'd have a comparable API surface.